samcli/commands/build/command.py

```diff
@@ -275,6 +287,7 @@ def do_cli(  # pylint: disable=too-many-locals, too-many-statements
         aws_region=click_ctx.region,
         hook_name=hook_name,
         build_in_source=build_in_source,
+        mount_with=mount_with,
```

Comment:
I would set the default (READ) here in case it is `None` to make the rest of the implementation easier (we don't need to check if it is `None` or any other value.)